packages/storage/amplify_storage_s3_dart/pubspec.yaml

```diff
@@ -30,5 +31,6 @@ dev_dependencies:
   build_verify: ^3.0.0
   built_value_generator: 8.4.2
   drift_dev: ^2.2.0+1
+  json_serializable: ^6.5.4
```

Comment:
Should be pinned since updating it potentially changes the generated code and we track that. Can run `aft deps apply` to get the latest.